contentstack_utils.gemspec

```diff
@@ -21,8 +21,8 @@ Gem::Specification.new do |s|
   s.test_files    = s.files.grep(%r{^spec/})
   s.require_paths = ["lib"]
 
-  s.add_dependency 'activesupport', '>= 3.2', '< 7.0.4'
-  s.add_dependency 'nokogiri', '~> 1.11', '>= 1.11.0'
+  s.add_dependency 'activesupport', '>= 3.2'
```

Comment:
@sunil-lakshman please make this similar to delivery SDK:
`'activesupport', '~> 3.2'`

Reply:
Done